readthedocs/templates/account/signup.html

```diff
@@ -0,0 +1,33 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Sign up" %}{% endblock %}
+
+{% block body_class %}signup-page{% endblock %}
+
+{% block content %}
+<h1>{% trans "Sign Up" %}</h1>
+
+<p>
+  <small>{% blocktrans %}Already have an account? Then please <a href="{{ login_url }}">sign in</a>.{% endblocktrans %}</small>
+</p>
+
+<form class="signup" id="signup_form" method="post" action="{% url 'account_signup' %}">
+  {% csrf_token %}
+  {{ form.as_p }}
+  {% if redirect_field_value %}
+  <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
+  {% endif %}
+  <button type="submit">{% trans "Sign Up" %} &raquo;</button>
+</form>
+
+<h3>{% trans 'Or' %}</h3>
+
+<div class="clearfix">
+  <ul class="socialaccount_providers">
+    {% include "socialaccount/snippets/provider_list.html" with process="login" next="" verbiage="Sign up with" %}
```

Comment:
`process="login"` and `verbiage="Sign up with"`

Just in case, `process` shouldn't be `signup` or similar?

Reply:
`login` is correct, I believe. I tested the full workflow.